packages/generator/tests/test_assets/example_renders/measurement_plugin_client/test_measurement_client.py

```diff
@@ -0,0 +1,398 @@
+"""Python Measurement Plug-In Client."""
+
+import logging
+import threading
+from pathlib import Path
+from typing import List, NamedTuple, Optional
+
+import grpc
+from google.protobuf import any_pb2
+from google.protobuf import descriptor_pool
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.measurementlink.measurement.v2 import (
+    measurement_service_pb2 as v2_measurement_service_pb2,
+    measurement_service_pb2_grpc as v2_measurement_service_pb2_grpc,
+)
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import (
+    DoubleXYData,
+)
+from ni_measurement_plugin_sdk_service.discovery import DiscoveryClient
+from ni_measurement_plugin_sdk_service.grpc.channelpool import GrpcChannelPool
+from ni_measurement_plugin_sdk_service.measurement.client_support import (
+    create_file_descriptor,
+    deserialize_parameters,
+    ParameterMetadata,
+    serialize_parameters,
+)
+
+_logger = logging.getLogger(__name__)
+
+_V2_MEASUREMENT_SERVICE_INTERFACE = "ni.measurementlink.measurement.v2.MeasurementService"
+
+
+class Output(NamedTuple):
+    """Measurement result container."""
+
+    float_out: float
+    double_array_out: List[float]
+    bool_out: bool
+    string_out: str
+    string_array_out: List[str]
+    path_out: Path
+    path_array_out: Path
```

Comment:
Type-hinting is incorrect. Refer to my other comment that highlights the miss of path array handling.